documentation/roadmap.md

```diff
@@ -0,0 +1,39 @@
+# Perseo CEP Roadmap
+
+This product is an Incubated FIWARE Generic Enabler. If you would like to learn about the overall Roadmap of FIWARE, please check section "Roadmap" on the FIWARE Catalogue.
+
+## Introduction
+
+This section elaborates on proposed new features or tasks which are expected to be added to the product in the foreseeable future. There should be no assumption of a commitment to deliver these features on specific dates or in the order given. The development team will be doing their best to follow the proposed dates and priorities, but please bear in mind that plans to work on a given feature or task may be revised. All information is provided as a general guidelines only, and this section may be revised to provide newer information at any time.
+
```

Comment:
You could add also the following disclaimer (https://github.com/telefonicaid/fiware-orion/blob/master/doc/roadmap.md#introduction):

* This section has been last updated in January 2019. Please take into account its content could be obsolete.
* Note we develop this software in Agile way, so development plan is continuously under review. Thus, this roadmap has to be understood as rough plan of features to be done along time which is fully valid only at the time of writing it. This roadmap has not be understood as a commitment on features and/or dates.
* Some of the roadmap items may be implemented by external community developers, out of the scope of GEi owners. Thus, the moment in which these features will be finalized cannot be assured.

Up to you :). If you prefer not adding it and keep it as it is now, then NTC on this comment.

Reply:
Fixed in 8a43e97